src/components/SetSlippageModal/SetSlippageModal.tsx

```diff
@@ -0,0 +1,123 @@
+import { useCallback, useEffect } from "react";
+import { Settings } from "react-feather";
+import { NumericalInput } from "../NumericalInput/NumericalInput";
+import { useTradeActionHandlers, useTradeState } from "../../state/trade/hooks";
+import { DefaultTxnSettings } from "../../state/trade/actions";
+import { Box, Flex, Text } from "@radix-ui/themes";
+import { theme } from "../../theme/theme";
+import { MINIMUM_SLIPPAGE_VALUE } from "../../pages/Trade/Trade";
+import Modal from "../Modal/Modal";
+
+export default function SetSlippageModal() {
+  const account = "";
+  const { slippageValue } = useTradeState();
+  const { onSetSlippage } = useTradeActionHandlers();
+
+  const handleResetSlippage = useCallback(
+    (e: any) => {
```

Comment:
do you need to declare the `e: any` input? can you remove as it's not used after?